tests/cluster/etl/test_upsertloader.py

```diff
@@ -5,14 +5,12 @@
 from atc.config_master import TableConfigurator
 from atc.delta import DbHandle, DeltaHandle
 from atc.etl.loaders.UpsertLoader import UpsertLoader
-from atc.etl.loaders.UpsertLoaderParameters import UpsertLoaderParameters
 from atc.utils import DataframeCreator
 from tests.cluster.delta import extras
 from tests.cluster.delta.SparkExecutor import SparkSqlExecutor
 
 
 class UpsertLoaderTests(DataframeTestCase):
```

Comment:
I don't see the need for test_02_can_perform_full_over_existing.
It should be possible to do and incremental and merge test from the state from test_01_can_perform_incremental_on_empty.

This would also make the data objects data1 to data4 a bit simpler, and therefore also have a simpler data5 object after the merge

Reply:
I have removed test 02. And cleaned up the test. Then I have added a test of the save many.